stackgl_modules/package.json

```diff
@@ -58,5 +58,16 @@
     "node-polyfill-webpack-plugin": "3.0.0",
     "webpack": "5.90.3",
     "webpack-cli": "5.1.4"
+  },
+  "overrides": {
```

Comment:
I don't think we need to have `overrides`.
Could you please revert the changes to `package.json` and `packages-lock.json` then 
inside `stackgl_modules` install these:
```sh
npm install https://github.com/hborchardt/gl-axes3d#83caafc058b2d60b6006e26432967c411c125fe1
npm install https://github.com/hborchardt/gl-cone3d#e9728cd068e0dfe11b92c5785974cd463f78c01c
npm install https://github.com/hborchardt/gl-error3d#ee55620d17f3cc7dfb104823c904f5070aba79ed
npm install https://github.com/hborchardt/gl-line3d#27123949f65e2d8deab6a6e131030073ed9cdf02
npm install https://github.com/hborchardt/gl-mesh3d#be2c7fcb68e70e001bbfb8021380d0d7ac75b1c9
npm install https://github.com/hborchardt/gl-scatter3d#086177000f405bc72f171350f9c1efa6eb523e29
npm install https://github.com/hborchardt/gl-streamtube3d#29c914cc0d88fc5dffce62dcd8169f38115d078b
npm install https://github.com/hborchardt/gl-surface3d#0728e33c0296c602b147e8f1ea8f1bfe7040c61e
```

Reply:
Okay, but in that case I need to add 
`npm install https://github.com/hborchardt/gl-spikes3d#9ad06b2dd48fa19d2faa54e5645c2e4bf5e6157c`
as well, because that contains the pattern but is only indirectly depended on via e.g. gl-mesh3d